updating/updating-cluster-within-minor.adoc

```diff
@@ -18,6 +18,7 @@ Use the web console or `oc adm upgrade channel _<channel>_` to change the update
 * Have access to the cluster as a user with `admin` privileges.
 See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
 * Have a recent xref:../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
+* Support for {op-system-base}7 workers are removed in {product-version}. You must replace {op-system-base}7 workers with {op-system-base}8 or {op-system} workers before upgrading to {product-version}. Red Hat does not support in place {op-system-base}7 to {op-system-base}8 upgrades for {op-system-base} workers, those hosts must be replaced with a clean OS install.
```

Comment:
* are removed -> is removed
* You should add "{product-title}" in front of where you say "{product-version}" (2 places)
* in place -> in-place ?
* The last sentence should use a semicolon instead of a comma
* I'd write out OS -> "operating system"

Same for the other modules where this is copy/pasted.